/**:
  ros__parameters:
    update_rate: 10.0
    system_emergency_heartbeat_timeout: 0.5
    external_emergency_stop_heartbeat_timeout: 0.0
    vel_lim: 25.0
    lon_acc_lim: 5.0
    lon_jerk_lim: 5.0
    lat_acc_lim: 5.0
    lat_jerk_lim: 5.0
